bindings/wasm/manifold.d.ts

```diff
@@ -37,10 +38,10 @@ export interface ManifoldStatic {
   setCircularSegments: typeof T.setCircularSegments;
   getCircularSegments: typeof T.getCircularSegments;
   reserveIDs: typeof T.reserveIDs;
+  CrossSection: typeof T.CrossSection;
   Manifold: typeof T.Manifold;
   Mesh: typeof T.Mesh;
   setup: () => void;
 }
 
-export default function Module(config?: {locateFile: () => string}):
-    Promise<ManifoldStatic>;
+export default function Module(config: { locateFile: () => string }): Promise<ManifoldStatic>;
```

Comment:
Intensional?

Reply:
That'd be autoformatting

Reply:
No, you lost a `?` I added in my last PR.

Reply:
Oh then it's my fault when I was resolving merge conflicts.